docs/docs/forms.md

```diff
@@ -76,6 +76,8 @@ handleChange(event) {
 }
 ```
 
+However, if `this.state.value` becomes `null` or `undefined` at any time (including the initial state), React will evaluate the component as uncontrolled and may produce a warning: `Input elements should not switch from uncontrolled to controlled (or vice versa)`.
+
```

Comment:
ok, I got my help with this particular warning on stack overflow and there are no other objections to your position. I will update my PR to use your suggestion.